src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -37,6 +37,14 @@
     </ItemGroup>
   </Target>
 
+  <ItemGroup Label="NuGet Shared">
+    <Compile Include="$(SharedDirectory)\XmlUtility.cs" />
```

Comment:
Is it worth sorting these?

Reply:
Whoops!   When I was creating the PR, I felt like there was something I wanted to do but forgot, and this is it! 🤦